ocpp-v1_6/src/test/java/eu/chargetime/ocpp/model/test/SetChargingProfileConfirmationTest.java

```diff
@@ -0,0 +1,44 @@
+package eu.chargetime.ocpp.model.test;
+
+import eu.chargetime.ocpp.model.core.IdTagInfo;
+import eu.chargetime.ocpp.model.core.StartTransactionConfirmation;
+import eu.chargetime.ocpp.model.smartcharging.ChargingProfileStatus;
+import eu.chargetime.ocpp.model.smartcharging.SetChargingProfileConfirmation;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
+import static org.mockito.Mockito.mock;
+
+public class SetChargingProfileConfirmationTest {
+    SetChargingProfileConfirmation confirmation;
+
+    @Before
+    public void setUp() throws Exception {
+        confirmation = new SetChargingProfileConfirmation();
+    }
+
+    @Test
+    public void setIdTagInfo_aChargingProfileStatus_statusIsSet() {
```

Comment:
I probably did this myself as well, but there is no need to test POJO methods if they don't have any validation. No harm though.